stories/SQFormReadOnlyField.stories.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import {Story, Meta} from '@storybook/react';
 
 import {SQFormReadOnlyField as SQFormReadOnlyFieldComponent} from '../src';
+import type {SQFormReadOnlyFieldProps} from 'components/SQForm/SQFormReadOnlyField';
 import {SQFormStoryWrapper} from './components/SQFormStoryWrapper';
 import {createDocsPage} from './utils/createDocsPage';
 import getSizeProp from './utils/getSizeProp';
-import markdown from '../notes/SQFormReadOnlyField.md';
+import type {gridOptions} from './utils/getSizeProp';
```

Comment:
Is there any particular reason why this type is not capitalized?
I am happy to update it here.

Reply:
> Is there any particular reason why this type is not capitalized? I am happy to update it here.

Yea that's be nice. It's also a little odd importing a `type` called `gridOptions` from a utils folder named `getSizeProp`. I don't think this type would be very discoverable.

Might be worth moving to a `types` folder

Reply:
I believe we made in the _very early_ TS adoption process. We were just babies back then 🍼